modules/channel-web/src/views/lite/components/messages/MessageGroup.tsx

```diff
@@ -64,37 +67,35 @@ class MessageGroup extends React.Component<Props> {
       <div
         role="main"
         className={classnames('bpw-message-big-container', {
-          'bpw-from-user': !this.props.isBot,
-          'bpw-from-bot': this.props.isBot
+          'bpw-from-user': !isBot,
+          'bpw-from-bot': isBot
         })}
       >
-        {this.props.avatar}
+        {avatar}
         <div role="region" className={'bpw-message-container'}>
-          {this.props.showUserName && <div className={'bpw-message-username'}>{this.props.userName}</div>}
+          {showUserName && <div className={'bpw-message-username'}>{userName}</div>}
           <div aria-live="assertive" role="log" className={'bpw-message-group'}>
             <span data-from={fromLabel} className="from hidden" aria-hidden="true">
               {fromLabel}
             </span>
-            {this.props.messages.map((data, i) => {
-              const isLastMsg = i == this.props.messages.length - 1
-              const payload = this.convertPayloadFromOldFormat(data)
+            {sortBy(messages, 'eventId').map((message, i, messages) => {
```

Comment:
Would this mess up ordering of messages which have their eventId set to null? What do you think of using `orderBy(messages, ['sent_on', 'eventId'])`  instead ?

Reply:
it wont mess up no, it keeps the same order. I wrote your solution first but then realized that: 


```js
const data = [{a:1},{a:2}]
_.sortBy(data, 'eventId') // yields ==> [{a:1},{a:2}]
```